src/main/java/org/folio/service/orders/flows/update/unopen/UnOpenCompositeOrderManager.java

```diff
@@ -240,37 +235,6 @@ private Future<List<Pair<String, String>>> deleteHoldingsByItems(List<JsonObject
       });
   }
 
-  private Future<List<Pair<String, String>>> deleteHoldings(List<JsonObject> holdings, RequestContext requestContext) {
```

Comment:
We should not delete this method, because we can have holding that not has any items and we can choose to delete this holding or keep it, so we should remain this method.
Scenario 3 from this acceptance criteria: https://issues.folio.org/browse/UIREC-236

Reply:
Probably we should add some new parameter, because with deleteHoldingAndItems we can resolve only  Synchronized flow to support these buttons from UI:
- Cancel
- Delete Holdings and items
- Delete items

To resolve independent flow proably we need to add additional param, because we need to support actions for these buttons:
- Cancel
- Keep Holdings
- Delete Holdings

Reply:
fixed